src/h3/api/_api_template.py

```diff
@@ -136,24 +178,65 @@ def h3_distance(h1, h2):
         return d
 
     def h3_to_geo_boundary(h, geo_json=False):
+        """ Return tuple of lat/lng paris describing
+        the cell boundary
+
+        If `geo_json` is True, return lng/lat pairs
```

Comment:
We should remark on what the ordering is if this is false.

Reply:
updated